hail/Makefile

```diff
@@ -12,6 +12,7 @@ SPARK_VERSION=2.4.0
 HAIL_PIP_VERSION=0.2.14
 
 HAIL_PYTHON3?=python3
+HAIL_PIP3?=pip3
```

Comment:
I found that installing pip3 sometimes overwrites pip (and doesn't always install pip3?  Not sure.) and the safe thing seems like `python3 -m pip ...`.  Also, spaces?

Reply:
Also, why HAIL_PYTHON3 here, but just PYTHON elsewhere?